.github/CODEOWNERS

```
@@ -26,6 +26,7 @@
 /examples/gno.land/p/demo/avl/        @jaekwon
 /examples/gno.land/p/demo/bf/         @moul
 /examples/gno.land/p/demo/blog/       @gnolang/devrels
+/examples/gno.land/p/demo/boardsv2/   @ilgooz @jeronimoalbi
```

also shouldn't this be /boards/v2?

jeronimo needs to be added with write access to the repo if he is to be a codeowner.

---

@moul can you give jeronimoalbi a write access?

@thehowl the name could be changed, we're not actually making a v2 of boards. Potentially boardsv2 will be named just boards. We can figure this out later.

---

```suggestion
/examples/gno.land/p/demo/boardsv2/   @ilgooz @jeronimoalbi @moul
```

Let's add both of you as code owners of this directory. I will also add myself since I am ultimately responsible for this package on behalf of the core team.